glue_jupyter/bqplot/common/tools.py

```diff
@@ -19,12 +19,27 @@ def __init__(self, viewer):
         self.viewer = viewer
 
     def activate(self):
+
+        # Disable any active tool in other viewers
+        for viewer in self.viewer.session.application.viewers:
+            if viewer is not self.viewer:
+                viewer.toolbar.active_tool = None
+
```

Comment:
I moved this to the base class for all the tools as that then matches what is in the Qt glue package